src/tests/ftest/util/ior_test_base.py

```diff
@@ -96,14 +92,34 @@ def create_pool(self):
         # Create a pool
         self.pool.create()
 
+    def _create_cont(self):
+        """Create a container.
+
+        Returns:
+            str: UUID of the created container
+
+        """
+        cont_type = self.params.get("type", "/run/container/*")
+        result = self.daos_cmd.container_create(
+            pool=self.pool.uuid, svc=self.pool.svc_ranks,
+            cont_type=cont_type)
+
+        # Extract the container UUID from the daos container create output
+        cont_uuid = re.findall(
+            "created\s+container\s+([0-9a-f-]+)", result.stdout)
```

Comment:
(lint) '\s'. String constant might be missing an r prefix.
(lint) '\s'. String constant might be missing an r prefix.